README.md

```diff
@@ -159,6 +159,7 @@ bucket.
 
 | Version | Cloud TPU VMs Wheel |
 |---------|-------------------|
+| 2.1 (Python 3.8) | `https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torch_xla-2.1-cp38-cp38-linux_x86_64.whl` |
```

Comment:
If you're going to move this into the `details` zipper, I think also move the 2.1 XRT build.